packages/website/ts/containers/connect_documentation.tsx

```diff
@@ -0,0 +1,97 @@
+import BigNumber from 'bignumber.js';
+import * as _ from 'lodash';
+import * as React from 'react';
+import {connect} from 'react-redux';
+import {Dispatch, Store as ReduxStore} from 'redux';
+import {Blockchain} from 'ts/blockchain';
+import {DocsInfo} from 'ts/pages/documentation/docs_info';
+import {
+    Documentation as DocumentationComponent,
+    DocumentationAllProps,
+} from 'ts/pages/documentation/documentation';
+import {Dispatcher} from 'ts/redux/dispatcher';
+import {State} from 'ts/redux/reducer';
+import {DocsInfoConfig, WebsitePaths} from 'ts/types';
+import {typeDocUtils} from 'ts/utils/typedoc_utils';
+
+/* tslint:disable:no-var-requires */
+const IntroMarkdown = require('md/docs/connect/introduction');
+const InstallationMarkdown = require('md/docs/connect/installation');
+/* tslint:enable:no-var-requires */
+
+const connectDocSections = {
+    introduction: 'introduction',
+    installation: 'installation',
+    httpClient: 'httpClient',
+    types: 'types',
+};
+
+const docsInfoConfig: DocsInfoConfig = {
+    packageName: '0x Connect',
```

Comment:
is this supposed to be 0x.js?

Reply:
Let me work on the naming here since both you and Leo got confused about this.